doc/DeveloperGuide.md

```diff
@@ -88,6 +88,38 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to list tags
+2. AddressBook show the list of tags
+3. User requests to rename a specific tag in the list
```

Comment:
Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).